lib/scraper.js

```diff
@@ -26,17 +26,14 @@ function Query(qo) {
 Query.prototype.url = function() {
```

Comment:
Consider using a well-tested library for building the url or writing a more exhaustive algorithm.